composer.json

```diff
@@ -12,21 +12,24 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
+        "php": "^7.1",
         "symfony/framework-bundle": "^2.8|^3.0",
-        "sonata-project/seo-bundle": "^1.1.3|^2.0",
-        "doctrine/collections": "^1.0"
+        "sonata-project/seo-bundle": "^2.2",
+        "doctrine/collections": "^1.0",
+        "symfony/symfony": "^2.8",
+        "phpcr/phpcr-utils": "^1.2.9"
```

Comment:
did we start depending on this? does this make sense? do we not either depend on phpcr-odm or on nothing phpcr? if this really belongs here, then we don't need the conflicts section and can just do `^1.3.0` here.

Reply:
we didn't had it here, i put it there to get a better version, so i will clean it up, i even pushed the requirement on symfony.

Reply:
I even moved sebastian/exporter to conflict